core/src/apps/cardano/sign_tx.py

```diff
@@ -573,15 +563,19 @@ async def _show_standard_tx(
     for withdrawal in msg.withdrawals:
         await confirm_withdrawal(ctx, withdrawal)
 
-    has_metadata = bool(msg.metadata)
+    await show_auxiliary_data(
```

Comment:
also the pool registration tx can contain auxiliary data if I'm not mistaken, so this should be done there as well (the logic missing there is therefore likely a bug, unless I forgot something)

Reply:
Nice. True. I missed that. Added in https://github.com/vacuumlabs/trezor-firmware/pull/57/commits/02a3b6f62c4f15bba6d5ed2f6ef0be7c19ee17e6. I also added a test case for such a transaction.

I've added it after the network confirmation screen and before the final confirmation.